### ISSUE TITLE
[Server] Add /stop command

### ISSUE THREAD

**Issue by specialkk** [MCMrARM/mcpelauncher-linux#328]
This would be neat since some web-based server panels expect to be able to shut down servers by passing in a text command. I could probably patch in a workaround myself to send an interrupt signal, but ideally we could have behavior that closely matches a Java edition server anyway.

This is awesome work by the way, keep doing what you've been doing!

**Comment by MCMrARM** [comment]
Hello,
this is not within the scope of the launcher app itself. I plan to do a serverutils mod in the future that adds a few command (/stop, whitelist, bans) in the future but that's not something I want to do short term.

What you could do is to 'patch' the command in to the current codebase, if all you want is to having it working in console and not in-game.

**Comment by codehz** [comment]
I think it can be a mod api.

**Comment by codehz** [comment]
Oh, I think /stop command can be implemented by sending a signal to self pid..

**Comment by nicman23** [comment]
does sigterm stop the server gracefully?

**Comment by codehz** [comment]
@nicman23 yes, you can send SIGINT(Ctrl+C) and it will be stopped gracefully
